tools/notcurses-ffi.pc.in

```diff
@@ -0,0 +1,14 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=@CMAKE_INSTALL_PREFIX@
+libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
+includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
+
+Name: @PROJECT_NAME@
+Description: TUI library for modern terminal emulators (ffi support)
+Version: @PROJECT_VERSION@
+
+Requires: notcurses-ffi
+Requires.private:
+Libs: -L${libdir} -lnotcurses-ffi
+Libs.private: -lstdc++
```

Comment:
is stdc++ really necessary? i don't see why it would be.

Reply:
👍